content/fr/docs/setup/custom-cloud/kubespray.md

```diff
@@ -0,0 +1,124 @@
+
+* Un cluster en haute disponibilité
+* des composants modulables
+* Le support des principales distributions Linux:
+  * Container Linux de CoreOS
+  * Debian Jessie, Stretch, Wheezy
+  * Ubuntu 16.04, 18.04
+  * CentOS/RHEL 7
+  * Fedora/CentOS Atomic
+  * openSUSE Leap 42.3/Tumbleweed
+* des tests d'intégration continue
+
+Afin de choisir l'outil le mieux adapté à votre besoin, veuillez lire [cette comparaison](https://github.com/kubernetes-incubator/kubespray/blob/master/docs/comparisons.md) avec [kubeadm](/docs/admin/kubeadm/) et [kops](../kops).
+
+{{% /capture %}}
+
+{{% capture body %}}
+
+## Créer un cluster
+
+### (1/5) Prérequis
+
+Les serveurs doivent être installés en s'assurant des éléments suivants:
+
+* **Ansible v2.6 (ou version plus récente) et python-netaddr installés sur la machine qui exécutera les commandes Ansible**
+* **Jinja 2.9 (ou version plus récente) est nécessaire pour exécuter les playbooks Ansible**
+* Les serveurs cibles doivent avoir **accès à Internet** afin de télécharger les images Docker. Autrement, une configuration supplémentaire est nécessaire, (se référer à [Offline Environment](https://github.com/kubernetes-sigs/kubespray/blob/master/docs/downloads.md#offline-environment))
+* Les serveurs cibles doivent être configurés afin d'autoriser le transfert IPv4 (**IPv4 forwarding**)
+* **Votre clé ssh doit être copiée** sur tous les serveurs faisant partie de votre inventaire Ansible.
+* La configuration du **pare-feu n'est pas gérée**. Vous devrez vous en charger en utilisant votre méthode habituelle. Afin d'éviter tout problème pendant l'installation nous vous conseillons de le désacttiver.
+* Si Kubespray est exécuté avec un utilisateur autre que "root", une méthode d'autorisation appropriée devra être configurée sur les serveurs cibles (exemple: sudo). Il faudra aussi utiliser le paramètre `ansible_become` ou ajouter `--become` ou `b` à la ligne de commande.
+
+Afin de vous aider à préparer votre de votre environnement, Kubespray fourni les outils suivants:
+
+* Scripts [Terraform](https://www.terraform.io/) pour les fournisseurs de cloud suivants:
+  * [AWS](https://github.com/kubernetes-incubator/kubespray/tree/master/contrib/terraform/aws)
+  * [OpenStack](https://github.com/kubernetes-incubator/kubespray/tree/master/contrib/terraform/openstack)
+
+### (2/5) Construire un fichier d'inventaire Ansible
+
+Lorsque vos serveurs sont disponibles, créez un fichier d'inventaire Ansible ([inventory](http://docs.ansible.com/ansible/intro_inventory.html)).
+Vous pouvez le créer manuellement ou en utilisant un script d'inventaire dynamique. Pour plus d'informations se référer à [Building your own inventory](https://github.com/kubernetes-incubator/kubespray/blob/master/docs/getting-started.md#building-your-own-inventory).
+
+### (3/5) Préparation au déploiement de votre cluster
+
+Kubespray permet de personnaliser de nombreux éléments:
+
+* Choix du mode: kubeadm ou non-kubeadm
```

Comment:
i thought that in last kubespray version default mode is kubeadm, (non-kubadm still supported?).
not an expert, but we should check :)

Reply:
non-kubeadm mode will be dropped soon but is still supported in 2.8.